src/jmcomic/jm_feature.py

```
@@ -0,0 +1,194 @@
+        """
+        根据注册来源推导执行时机：
+        download_album → after_album, download_photo → after_photo
+        """
+        if feature_from == 'download_album':
+            return when == 'after_album'
+        elif feature_from == 'download_photo':
+            return when == 'after_photo'
+        return False
+
+    def __call__(self, **kwargs):
+        """带自定义参数，返回新实例（继承默认参数）"""
```

Ruff (0.15.12)

[warning] Docstring contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF002)